instrumentation/akka-http-10.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkahttp/client/HttpExtClientInstrumentation.java

```diff
@@ -56,15 +57,14 @@ public static void methodEnter(
       with way of continuing to reusing it.
        */
       Context parentContext = currentContext();
-      if (!tracer().shouldStartSpan(parentContext)) {
+      if (request == null || !instrumenter().shouldStart(parentContext, request)) {
```

Comment:
if null `request` is not valid input for the library method anyways, we typically (for better or worse?) don't validate the input and let the advice method throw NPE (which is then suppressed by `suppress = Throwable.class`)